app/models/user_preference.rb

```diff
@@ -0,0 +1,10 @@
+class UserPreference < ActiveRecord::Base
+
+  belongs_to :user
+  validates :name, :value, presence: true
+
+  OPTIONS = {
+    "Facility Home Page" => ["Dashboard", "Timeline View"],
```

Comment:
Hmm. This option feels like it belongs in UIC/dashboard engine. Not quite sure the best way to get it there.